### ISSUE TITLE
Different accounts, Different layouts 

### ISSUE THREAD

**Issue by Paramedic1978** [tullamods/Bagnon#436]
My alt accounts seem to have a different bag layout than my main account.  The backpack seems to be the last bag, and the first and last bag slots are reversed also.  Changing things in frame settings just seem to make it worse.  Any help is appreciated!

**Comment by Paramedic1978** [comment]
![wowscrnshot_052516_230320](https://cloud.githubusercontent.com/assets/19482793/15564414/5c214e28-22cf-11e6-9c0f-c935d3100d83.jpg)

**Comment by Paramedic1978** [comment]
![wowscrnshot_052516_230334](https://cloud.githubusercontent.com/assets/19482793/15564418/666c3b5e-22cf-11e6-99c2-9b28334dde8d.jpg)

**Comment by Paramedic1978** [comment]
Hopefully the developer can see what I am speaking about from these two screenshots.

**Comment by Tuzark** [comment]
I am having the exact same problem as of recently, but only with new alts it seems. On new alts it's like you said, it's like the backpack starts at the bottom/near bottom, so whenever I sort it always starts by stacking items on the second lowest row. And just like you, I have the exact same settings on ALL my characters, yet it only happens to new alts, even if they have the same settings.

I'll upload some screenshots as well. This is one of my older alts, who has been using Bagnon for a few years, when I use the sort function on his bags it looks like this: (as it does on all my older characters)
![bagnonproblem1](https://cloud.githubusercontent.com/assets/19935182/16052627/59bbb768-3265-11e6-84e0-a97b65dccab2.png)

However on my new alts, it looks like this:
![bagnonproblem2](https://cloud.githubusercontent.com/assets/19935182/16052722/bcb970d0-3265-11e6-8eaa-bf4cfdb830ba.png)

As you can see here it looks like the backpack starts where the hearthstone is, which is on the second-lowest row, and when I press sort, it sorts from where the heartstone is and downwards, until all the rows below are full, then it puts stuff on the row above the hearthstone.. Also even if I fill up like 2 more rows above, the sorting will for some reason avoid putting items in the 4 slots left of the hearthstone. xD

Hopefully this will help in fixing it, because this is an annoying problem that I'd rather not have on my Demon Hunter when Legion comes. :P

**Comment by Xodium** [comment]
I have the exact same problem, however I do not see the:
Game Menu > Interface > Game > Controls "Reverse clean up bags" option.

**Comment by Tuzark** [comment]
Yeah this option was removed with the Legion Pre-Expansion patch, like a lot of other interface options. :(

**Comment by Xodium** [comment]
I see, so since this issue is flagged as a game issue, does anyone know a way around it to fix the issue?
